gcn/notices/neutrino/Alert.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/neutrino/Alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "NeutrinoAlert",
+  "description": "Neutrino alert information common to all neutrino telescope alerts",
```

Comment:
high-energy neutrinos, and also make clear this is not a topic on it's own, but inherited by others

Reply:
base schema

Reply:
Done